Docs (#76)
Azure/azure-kusto-spark: docs/KustoSink.md

   This is the recommended option for typical use cases. However, using it results in blocking
 Spark driver for as long as the operation is running, up to several minutes. 

>  To avoid blocking Spark driver, it is possible to execute Kusto 'write' operation in an 

Reviewer: ```suggestion
 To **avoid blocking Spark driver**, it is possible to execute Kusto 'write' operation in an 
```

 opportunistic mode as an asynchronous operation, this is recommended only for spark streaming.
  The resulted behavior is the following:
   * Spark driver is not blocked

>    * In a success scenario, all data is written eventually only if the job is still running 

Reviewer: ```suggestion
   * In a success scenario, all data will eventually be written, only if the job is left running.  Job success status **DOES NOT** mean data is committed yet.
```

   This is an upper limit that may coexist with addition timeout limits as configured on Spark or Kusto clusters.  
   Default: '5400' (90 minutes)

* **KustoSinkOptions.KUSTO_SPARK_INGESTION_PROPERTIES_JSON**:
    A json representation of a SparkIngestionProperties (use toString to make a json of an instance).
    
    Properties:
        

>     dropByTags,,ingestByTags, additionalTags, ingestIfNotExists: util.ArrayList[String] - 

Reviewer: ```suggestion
    dropByTags,ingestByTags, additionalTags, ingestIfNotExists: util.ArrayList[String] - 
```

  .save()
``` 

IngestionProperties and short scala usage:

> ```

Reviewer: ```suggestion
```scala
```
